executor/infoschema_reader.go

```diff
@@ -32,6 +32,12 @@ import (
 	"github.com/pingcap/tidb/util/sqlexec"
 )
 
+const (
```

Comment:
It's better to keep them in `infoschema` package.

Reply:
Fixed, PTAL~